index.html

```diff
@@ -122,6 +122,33 @@ <h2>A project for first-time contributions</h2>
 
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Robin Shindelman </p>
+          <p class="contact">
+           <!--  <i class="fab fa-x-twitter"></i> -->
+           <!-- <a href="https://www.twitter.com" target="_blank">Your handle</a> -->
```

Comment:
Kindly note that if you are not pasting your profile links, then you are free to remove those lines instead of commenting. This is just for your information & learning and no modification is required from your end.